include/RAJA/pattern/launch/launch_core.hpp

```diff
@@ -294,20 +388,86 @@ RAJA::resources::Resource Get_Host_Resource(T host_res, RAJA::ExecPlace device){
 }
 #endif
 
-
-//Launch API which takes team resource struct
-template <typename POLICY_LIST, typename BODY>
+//Launch API which takes team resource struct and supports new reducers
+template <typename POLICY_LIST, typename ... ReduceParams>
 resources::EventProxy<resources::Resource>
-launch(RAJA::resources::Resource res, LaunchParams const &params, BODY const &body)
+launch(RAJA::resources::Resource res, LaunchParams const &launch_params,
+       const char *kernel_name, ReduceParams&&... rest_of_launch_args)
 {
-  return launch<POLICY_LIST>(res, params, nullptr, body);
+
+  //Get reducers
+  auto reducers = expt::make_forall_param_pack(std::forward<ReduceParams>(rest_of_launch_args)...);
+
+  auto&& launch_body = expt::get_lambda(std::forward<ReduceParams>(rest_of_launch_args)...);
+
+  ExecPlace place;
+  if(res.get_platform() == RAJA::Platform::host) {
+    place = RAJA::ExecPlace::HOST;
+  } else {
+    place = RAJA::ExecPlace::DEVICE;
+  }
+
+  //
+  //Configure plugins
+  //
+#if defined(RAJA_GPU_ACTIVE)
+  util::PluginContext context{place == ExecPlace::HOST ?
+      util::make_context<typename POLICY_LIST::host_policy_t>()
+      : util::make_context<typename POLICY_LIST::device_policy_t>()};
```

Comment:
```suggestion
      util::make_context<typename POLICY_LIST::device_policy_t>()};
```

Consistency between '?' and ':'